### ISSUE TITLE
Adding option to set reviewer

### ISSUE THREAD

**Issue by wouter-heerwegh** [releasehub-com/github-action-create-pr-parent-submodule#6]
Hi,

It would be nice to be able to specify a reviewer to the made pull request, as it would notify the needed people.

**Comment by nickdesaulniers** [comment]
+1, it looks like there's an API for adding "requested reviewers:" https://docs.github.com/en/rest/pulls/review-requests?apiVersion=2022-11-28#request-reviewers-for-a-pull-request

**Comment by wouter-heerwegh** [comment]
Yes I implemented it myself, but found the PRs a bit annoying. I'm now using dependabot, which works pretty well. I'm closing this issue as I'm no longer requesting this feature.
